lib/fca/import.rb

```diff
@@ -62,5 +62,9 @@ def import(file)
 
       [file, import_status, outcomes]
     end
+
+    def files_to_be_imported
```

Comment:
Moving this into its own method is great.

Just a note that it does make it harder to spot the difference when reviewing, though. :-/